keytree/factory.py

```diff
@@ -59,6 +59,13 @@ def geometry_Track(element, kmlns: dict = NSMAP):
     return Geometry("LineString", tuple(coords))
 
 
+def geometry_MultiTrack(element, kmlns: dict = NSMAP):
```

Comment:
@dericke what would you think about using `typing.Mapping` instead of `dict` in these cases? If I read the elementtree API docs right, the namespaces arg is only required to be a Mapping, and if we used that type, mypy (for example) could check that we're not modifying kmlns within the functions.

Reply:
I think that's a great idea, and went ahead and did so. I hadn't read too deeply into the typing module docs, so I wasn't aware that was even an option.